src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -405,7 +405,15 @@ public Index GetIndex(string name)
                 try
                 {
                     var (etag, _) = await _serverStore.SendToLeaderAsync(command);
-                    await _documentDatabase.RachisLogIndexNotifications.WaitForIndexNotification(etag);
+                    try
+                    {
+                        await _documentDatabase.RachisLogIndexNotifications.WaitForIndexNotification(etag , timeoutForIndexNotification);
+
+                    }
+                    catch (TimeoutException)
+                    {
```

Comment:
It should be ignored only if we recreating indexes

Reply:
No, it should never be ignored. What is the scenario where that is a good thing to do?